src/ServiceStack.OrmLite.Oracle.Tests/OracleParamTests.cs

```diff
@@ -269,328 +259,922 @@ public void ORA_ParamByteTest()
         }
 
         [Test]
-        public void ORA_ReservedNameComment_Test()
+        public void ORA_ReservedNameLevelAsAlias_Test()
         {
             using (var db = OpenDbConnection())
             {
                 DropAndCreateTables(db);
 
-                var row = new ParamComment { Comment = 1, Id = 2 };
+                var row = new ParamLevelAsAlias { Id = 2, ApprovalLevel = 212218 };
                 db.Insert(row);
 
-                row.Comment = 454;
+                row.ApprovalLevel = 676;
                 db.Update(row);
             }
         }
 
+        [ImportMany(typeof(IParam))]
```

Comment:
@TroyDycavinu-AI Yeah adding more deps makes it harder to port, e.g. future PCL profiles, iOS/Android, or CoreCLR builds. Extra deps can also create issues with peoples existing environments, e.g. devs using tools like SmartAssembly, ILMerge, etc. and using obscure parts of .NET Fx is also more likely to have issues running on Mono, which is why I generally prefer to run custom code where possible.

`System.ComponentModel` should already be referenced, do you know if `System.ComponentModel.Composition/Hosting` is in the same .dll?